nipype/interfaces/bru2nii.py

```diff
@@ -0,0 +1,54 @@
+
+"""The bru2nii module provides basic functions for dicom conversion
+
+	Change directory to provide relative paths for doctests
+	>>> import os
+	>>> filepath = os.path.dirname( os.path.realpath( __file__ ) )
+	>>> datadir = os.path.realpath(os.path.join(filepath, '../testing/data'))
+	>>> os.chdir(datadir)
+"""
+
+from nipype.interfaces.base import (CommandLine, CommandLineInputSpec, traits, TraitedSpec, isdefined, File, Directory)
+import os
+
+
+
+class Bru2InputSpec(CommandLineInputSpec):
+	input_dir = Directory(desc = "Input Directory", exists=True, mandatory=True, position=-1, argstr="%s")
+	actual_size = traits.Bool(argstr='-a', desc="Keep actual size - otherwise x10 scale so animals match human.")
+	force_conversion = traits.Bool(argstr='-f', desc="Force conversion of localizers images (multiple slice orientations).")
+	append_protocol_name = traits.Bool(argstr='-p', desc="Append protocol name to output filename.")
+	output_filename = traits.Str(argstr="-o %s", desc="Output filename ('.nii' will be appended)", genfile=True)
+
+class Bru2OutputSpec(TraitedSpec):
+	nii_file = File(exists=True)
+
+class Bru2(CommandLine):
+	"""Uses bru2nii's Bru2 to convert Bruker files
+
+	Examples
+	========
+
+	>>> from nipype.interfaces.bru2nii import Bru2
+	>>> converter = Bru2()
+	>>> converter.inputs.input_dir = os.getcwd()
```

Comment:
Use a relative path to an existing folder in "testing/data". For example "dicomdir". If this name does not make sense in this context you can create a new folder (and add an empty dummy file in it so git would register it).